plone/app/content/browser/contents/__init__.py

```diff
@@ -201,31 +227,13 @@ def get_columns(self):
             'Type': translate(_('Type'), context=self.request),
             'total_comments': translate(_('Total comments'), context=self.request),  # noqa
         }
-        # These columns either have alternatives or are probably not useful
-        ignored = [
-            'Date',
-            'Title',
-            'author_name',
-            'cmf_uid',
-            'commentators',
-            'created',
-            'effective',
-            'expires',
-            'getIcon',
-            'getId',
-            'getRemoteUrl',
-            'in_response_to',
-            'listCreators',
-            'meta_type',
-            'modified',
-            'portal_type',
-            'sync_uid'
-        ]
+        # Filter out ignored
+        columns = {k: v for k, v in columns.iteritems() if k not in self.ignored_columns}  # noqa
```

Comment:
please avoid ``# noqa``, in such cases you can just write:

```
        columns = {
            k: v for k, v in columns.iteritems() 
            if k not in self.ignored_columns
        }
```

This also increases readability